lib/ebaze_web/controllers/page_controller.ex

```diff
@@ -1,8 +1,11 @@
 defmodule EbazeWeb.PageController do
   use EbazeWeb, :controller
 
+  alias Ebaze.Auctions
+
   def index(conn, _params) do
-    render(conn, "index.html")
+    auctions = Auctions.list_auctions()
+    render(conn, "index.html", auctions: auctions)
```

Comment:
😁 Down the road (make a card) - add filtering so only OPEN auctions (not ended) show up...don't want people seeing junk they can't bid on!

Reply:
https://trello.com/c/FB1IiVk7